Core/bin/README.md

```diff
@@ -0,0 +1,73 @@
+Scripts
+==========
+
+Scripts modules first of all loads the environment variables by the `getenv.sh` script. Some of the most important variables that are needed are:
+
+- MA_ENV_PROPERTIES
+- MA_ENV_PROPERTIES
+- mango_paths_data
+
+Mainly this 3 variables specify the core for mango, where the source code and libraries are; as well as the data base files. All of them are properly know if Mango was set up by an installer, whenever we run a start_mango.sh all required variables must be figured out by this `getenv.sh` if the installetion was done like that.
+
+Another important thing to consider is that we also have a `mango.properties` file where we can change different values; normally this file is populated with lots of variables commented and also it contains a description of each variable value.
+
+Also there are other variables that can be useful to declare in case of deployments of other actions, this variables are:
+
+- mango_paths_data
+- keystore
+- MA_KEYSTORE
+- MA_KEYSTORE_PASSWORD
+- MA_KEY_PASSWORD
+- MA_KEY_ALIAS
+
+It is important to know that if needed we can customized the value from this variables by setting the value in the `mango.properties` or even in the `env.properties`.
```

Comment:
Remove all references to env.properties, this is no longer used

Reply:
I removed references for env.properties